docs/direct-manipulation.md

```diff
@@ -62,23 +62,23 @@ Composite components are not backed by a native view, so you cannot call `setNat
 
 ```SnackPlayer name=setNativeProps%20with%20Composite%20Components
 import React from 'react';
-import { Text, TouchableOpacity, View } from 'react-native';
+import {Text, TouchableOpacity, View} from 'react-native';
 
 const MyButton = (props) => {
-    return (
-      <View style={{marginTop: 50}}>
-        <Text>{props.label}</Text>
-      </View>
-    )
-}
+  return (
+    <View style={{marginTop: 50}}>
+      <Text>{props.label}</Text>
+    </View>
+  );
+};
 
-export default App = () => {
-    return (
-      <TouchableOpacity>
-        <MyButton label="Press me!" />
-      </TouchableOpacity>
-    )
-}
+export default (App = () => {
```

Comment:
Curly braces after `export default` are unnecessary (and a bit confusing). It will be nice to omit them if config allows to.

Reply:
I'm not seeing where we can add that: https://prettier.io/docs/en/options.html

Reply:
It's look like a ESLint/Prettier issue with this particular construction. Expo Snack editor also adds those brackets on autoformat. I'm not sure why though since `export` is an expression/directive not a function.